docs-ui/components/docsLinks.tsx

```diff
@@ -1,5 +1,6 @@
 import {withTheme} from '@emotion/react';
 import styled from '@emotion/styled';
+/* eslint-disable-next-line import/default */
```

Comment:
Do you know why eslint fails here?

Reply:
It's because `@storybook/addon-links/react` exports LinkTo as a node module (`module.exports = require('./dist/cjs/react');`) instead of the `export default` syntax. 

The correct way to import this would be `import * as LinkTo from …`. This fixes the ESLint error, but results in a different rendering error:

<img width="648" alt="Screen Shot 2021-09-23 at 4 05 58 PM" src="https://user-images.githubusercontent.com/44172267/134595248-110eaea7-3964-4aba-ad0c-2ff808e7dbe7.png">